src/Api/Billing/Models/Requests/UpdateClientOrganizationRequestBody.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace Bit.Api.Billing.Models.Requests;
+
+public class UpdateClientOrganizationRequestBody
```

Comment:
This is a rename from `UpdateProviderOrganizationRequestBody` to not confuse it with AC's version. I also added the data annotation attributes, but it is not related to the ticket's acceptance criteria.